dbt/models/model/model.vw_pin_shared_input.sql

```diff
@@ -384,7 +393,10 @@ SELECT
     vwpf.nearest_neighbor_2_pin10,
     vwpf.nearest_neighbor_2_dist_ft,
     vwpf.nearest_neighbor_3_pin10,
-    vwpf.nearest_neighbor_3_dist_ft
+    vwpf.nearest_neighbor_3_dist_ft,
+
+    -- Year should be listed last, for partitioning
+    uni.year AS meta_year
```

Comment:
Unsure if we need this comment or if it's self-evident, but I added it to be safe.

Reply:
Always good to keep it IMO.